felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1856,6 +2033,26 @@ func (m *bpfEndpointManager) applyPolicy(ifaceName string) error {
 	return err
 }
 
+func mergeAttachPoints(ap, ap6 attachPoint) attachPoint {
+	if aptcV4, v4ok := ap.(*tc.AttachPoint); v4ok {
+		if aptcV6, v6ok := ap6.(*tc.AttachPoint); v6ok {
+			if aptcV4 != nil && aptcV6 == nil {
+				return aptcV4
+			} else if aptcV6 != nil && aptcV4 == nil {
+				return aptcV6
+			}
```

Comment:
what if none of them is nil?

Reply:
Will handle this case, when turning on dual stack.